src/txmempool.h

```diff
@@ -716,6 +719,12 @@ class CTxMemPool
         return m_unbroadcast_txids;
     }
 
+    /** Returns if a txid is in the unbroadcast set */
+    const bool InUnbroadcast(const uint256& txid) const {
```

Comment:
Ok after over-thinking this for a few hours 😂 I'm going to change it to `IsUnbroadcastTx` which looks pretty similar to the others and hopefully feels natural.
I definitely think `entry.IsUnbroadcast()` would be ideal if it were a mempool entry method (I didn't do it that way because `m_unbroadcast_txids` belongs to the mempool). I just don't want it to sound like I'm saying the mempool is unbroadcast.